wiki/useful-windows-tips.md

```diff
@@ -0,0 +1,42 @@
+# Useful tips and "hacks" for Windows
+
+### Get more options when right-clicking
+
+When right-clicking on a file or folder, you can hold down shift to get more options, such as to "Copy as path" which lets you copy the path of the file.
+
+
+### Windows Master Control Panel shortcut
+
+The Windows Master Control Panel shortcut, often referred to as "God Mode", is a shortcut to access all control panel settings in one place.
+
+To create the shortcut, create a folder and name it
+```
+allSettings.{ED7BA470-8E54-465E-825C-99712043E01C}
+```
+The text in front of the dot has no effect.
+
+You can now open the folder to find all control panel settings.
+
+### Make Windows search faster
+
+You can make the Windows Search way faster, by enabling Indexing, which indexes the locations you select. In simple words, it essentially creates a list of all files, so that when you search for a file, Windows searches in that list instead.
+
+First, search for "Indexing Options" in the Start menu. Then, in the window that appears, click "Modify". Then, click "Show All Locations". Now, check all the locations that you want Windows to index. Finally, click Ok to confirm. Your files will now be indexed.
```

Comment:
adding images here with details steps would help the reader

Reply:
I didn't add them cuz i thought they would be a bit too much cuz that would be like 4 images.

Reply:
it'll be fine
just don't make them too long/wide

Reply:
sure
question tho, u can see my drives in the screenshots and i don't really have a VM rn to take a screenshot from a fresh windows installation. should i make a new vm real quick or is this fine 🤔
![image](https://github.com/Microsoft-community/Community-website/assets/109022579/cb79041f-0fc2-4a2b-898f-2b1954fdb3a4)

Reply:
maybe try windows sandbox

Reply:
> maybe try windows sandbox

i forgot that's even a thing lol ye i'll do that
update : crap it wants me to restart 😭

Reply:
@waitwhatActo windows sandbox doesn't like indexing apparently
![image](https://github.com/Microsoft-community/Community-website/assets/109022579/2c562485-fc45-4dc4-90a1-65e19fd2711a)

Reply:
"index is not running"